site/docs/v1/tech/apis/two-factor.adoc

```diff
@@ -147,6 +147,30 @@ include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[
 
 include::docs/v1/tech/apis/_two-factor-disable-request-parameters.adoc[]
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Disable Multi-Factor Authentication With a UserId
+[.endpoint]
+.URI
+--
+[method]#DELETE# [uri]#/api/user/two-factor/`{userId}`#
+--
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[]
+
+include::docs/v1/tech/apis/_two-factor-disable-request-body.adoc[]
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#jwt-authentication[icon:id-badge[type=fas]] Disable Multi-Factor Authentication With a UserId Using a JWT
+[.endpoint]
+.URI
+--
+[method]#DELETE# [uri]#/api/user/two-factor/`{userId}`#
```

Comment:
When using a JWT, you don't provide a `userId` on the URL - the JWT provides the `userId` by using the `sub` claim.

Reply:
👍